box-share-sdk/src/main/java/com/box/androidsdk/share/activities/BoxInviteCollaboratorsActivity.java

```diff
@@ -67,9 +67,18 @@ public boolean onPrepareOptionsMenu(Menu menu) {
     }
 
     @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
```

Comment:
I could not see the "SHOW_COLLABORATORS" activity setting this result.
So I assume that either: we are always updating the list here, or if the user change the list and press back, this does not update.
Can you please check ?